PixivConfig.py

```diff
@@ -14,6 +15,65 @@
 script_path = PixivHelper.module_path()
 
 
+class CustomSanitizer:
```

Comment:
better move it to separate file or merge it with pixivHelper as this is not configuration.

Reply:
hmm... okay, i will pull a new one in that case, this is kinda messy now anyways.